mindsdb/interfaces/jobs/scheduler.py

```diff
@@ -0,0 +1,98 @@
+import random
+import time
+
+import torch.multiprocessing as mp
+
+from mindsdb.utilities.config import Config
+from mindsdb.utilities.log import initialize_log
+from mindsdb.utilities import log
+from mindsdb.interfaces.storage import db
+
+from mindsdb.interfaces.jobs.jobs_controller import JobsExecutor
+
+mp_ctx = mp.get_context('spawn')
+
+logger = log.get_log('jobs')
+
+
+def scheduler_monitor(config):
+    while True:
+
+        logger.debug('Scheduler check timetable')
+        try:
+            check_timetable(config)
+        except (SystemExit, KeyboardInterrupt):
+            raise
+        except Exception as e:
+            logger.error(e)
+
+        # different instances should start in not the same time
+        time.sleep(20 + random.randint(1, 10))
```

Comment:
Shouldn't this also be configurable? I can envision local deployments where the user wants monitoring at the per-second level (or maybe fractions of a second!).

Reply:
I will add to config. But for now we can't run job more often than every minute. So precision is not about seconds now.